app/views/shared/_user_nav.html.haml

```diff
@@ -8,15 +8,21 @@
       .col-sm-12
         %ul.nav.navbar-nav
           %li= link_to 'Home', root_path
-          - if can? :manage, Category
+          - if can? :read, Category
             %li= link_to 'Categories', categories_path
-          - if can? :manage, Product
+          - if can? :read, Product
             %li= link_to 'Products', products_path
-          - if can? :manage, Menu
+          - if can? :read, Menu
             %li= link_to 'Menus', menus_path
-          - if can? :manage, User
+          - if can? :crud, User
             %li= link_to 'Users', users_path
         %ul.nav.navbar-nav.navbar-right
+          %li.dropdown
+            = link_to current_user.email, '', class: 'btn btn-secondary dropdown-toggle', aria: { expanded: false, haspopup: true }, data: { toggle: 'dropdown' }
+            .dropdown-menu{"aria-labelledby" => "dropdownMenuLink"}
```

Comment:
oops, ai bày kiểu này thế, hash => không khuyên dùng, nên là 
`.dropdown-menu{"aria-labelledby": "dropdownMenuLink"}`
hoặc
`.dropdown-menu{aria: {labelledby: "dropdownMenuLink"}}`
douple quote "" cũng ko khuyên dùng. chỉ dùng khi có in giá trị thông qua variable